src/applications/disability-benefits/2346/containers/OrderHistory.jsx

```diff
@@ -0,0 +1,228 @@
+import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { focusElement } from 'platform/utilities/ui';
+import {
+  getDLCAccessoriesData,
+  getDLCBatteryData,
+  // NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+  // getDLCSocksData,
+  updateDLCData,
+} from '../actions';
+
+class OrderHistory extends Component {
+  // NOTE: Using React state for checkboxes -@maharielrosario at 1/8/2020, 7:47:21 PM
+  // When discussed with other devs, the concensus was to have a stateful
+  // component to house our checkboxes' state. But all other state is in Redux.
+  // Is there an easy way to house individual checkbox state in Redux?
+
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+    this.props.getDLCBatteryData();
+    // NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+    // this.props.getDLCSocksData();
+    this.props.getDLCAccessoriesData();
+  }
+
+  createDLCUIData = dlcData => {
+    const dlcUIData = dlcData.map(dlcDataObj => {
+      let checkbox;
+      if (dlcDataObj.productGroup === 'ACCESSORIES') {
+        checkbox = (
+          <ErrorableCheckbox
+            className="vads-u-margin-left--4"
+            onValueChange={value => value}
+            id={dlcDataObj.id}
+          />
+        );
+      } else {
```

Comment:
Probably don't need the else if you assign checkbox with this value by default: https://github.com/department-of-veterans-affairs/vets-website/pull/11679/files#diff-0c0073b4d8e5ce14fde85624b1964f2bR29

Reply:
This page has been deleted.